viewing_party/party.py

```diff
@@ -0,0 +1,130 @@
+#wave 1
+def create_movie(movie_title,genre,rating):
+    create_movie_dict = {'title': movie_title, 'genre': genre, 'rating':rating}
+    
+
+    if movie_title and genre and rating:
+        movie_dict = {'title':movie_title, 'genre':genre, 'rating':rating}
+        return movie_dict
+    else:
+        return None
+
+def add_to_watched(user_data,movie_title):
+    watched_list = user_data['watched']
+    watched_list.append(movie_title)
+    return user_data
+
+def add_to_watchlist(user_data,movie_title):
+    add_watchlist = user_data['watchlist']
+    add_watchlist.append(movie_title)
+    return user_data
+
+def watch_movie(user_data,movie_title):
+    movies_watched = user_data['watched']
+    movies_to_watch = user_data['watchlist']
+
+    for movie in movies_to_watch:
+        if movie_title == movie['title']:
+            movies_watched.append(movie)
+            movies_to_watch.remove(movie)
+
+    return user_data
+
+
+def get_watched_avg_rating(user_data):
+    # avg_rating = 0.0
+    # rating_sum = 0
+    # for i in range (len(user_data["watched"])):
+    #     rating_sum += user_data["watched"][i]["rating"]
+    #     #calculating
+    # rating_sum/len(user_data["watched"])
+    total_rating = 0
+    if len(user_data["watched"]) == 0:
+        return 0
```

Comment:
nice job dealing with a potential zero division error. What's another way we could have raised this exception?